examples/e-commerce/src/app/app.component.html

```diff
@@ -38,20 +33,18 @@
       <div class="container-header">
         <h2>Filters</h2>
         <ais-clear-refinements
+          data-layout="desktop"
           resetLabel="Clear filters"
         ></ais-clear-refinements>
+
+        <ais-stats data-layout="mobile">
+          <ng-template let-state="state">
+            <strong>{{ state.nbHits }}</strong> results
```

Comment:
The text size is smaller than on the design on this widget.

Reply:
fb8e4c4